riotfile.py

```diff
@@ -2595,13 +2595,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="gunicorn",
             command="pytest {cmdargs} tests/contrib/gunicorn",
-            pkgs={"requests": latest},
+            pkgs={"requests": latest, "gevent": latest},
+            env={
+                "AGENT_VERSION": "latest",
+            },
             venvs=[
-                Venv(
```

Comment:
Gunicorn dropped support for python 2.x in [v20.0](https://docs.gunicorn.org/en/stable/2019-news.html#id6) not 19.10. 

If we want to drop py2.7 support for gunicorn we should we update our [minimum support version](https://ddtrace.readthedocs.io/en/latest/index.html#supported-libraries) to 20.0.